api/swagger-spec/v1.json

```diff
@@ -13235,7 +13235,7 @@
    },
    "v1.EmptyDirVolumeSource": {
     "id": "v1.EmptyDirVolumeSource",
-    "description": "EmptyDirVolumeSource is temporary directory that shares a pod's lifetime.",
+    "description": "EmptyDirVolumeSource represents an empty directory for a pod. Empty directory volumes support ownership management and SELinux relabeling.",
```

Comment:
Should we leave "temporary" there ? EmptyDir is meant to be temporary storage no ?